test/snippet/core/type_list/list_traits_concat.cpp

```diff
@@ -0,0 +1,11 @@
+#include <seqan3/core/type_list/traits.hpp>
+
+int main()
+{
+    using t = seqan3::type_list<int, float, bool>;
+    using t2 = seqan3::type_list<double, char, int>;
+    using t3 = seqan3::type_list<int, int>;
+
+    static_assert(std::same_as<seqan3::type_list<int, float, bool, double, char, int, int, int>,
```

Comment:
I know it is picky :see_no_evil: but can you switch the order: first the concat and then the result.